auth/src/main/java/com/firebase/ui/auth/ui/phone/SpacedEditText.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2015 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Modifications copyright (C) 2017 Google Inc
+ */
+
+package com.firebase.ui.auth.ui.phone;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.text.Editable;
+import android.text.Spannable;
+import android.text.SpannableStringBuilder;
+import android.text.style.ScaleXSpan;
+import android.util.AttributeSet;
+
+import com.firebase.ui.auth.R;
+
+/**
+ * This element inserts spaces between characters in the edit text and expands the width of the
+ * spaces using spannables.
+ * This is required since Android's letter spacing is not available until API 21.
+ */
+public final class SpacedEditText extends android.support.v7.widget.AppCompatEditText {
```

Comment:
nit: `AppCompatEditText` instead of `android.support.v7.widget.AppCompatEditText`